pkgs/by-name/im/immich-public-proxy/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nixosTests,
+  nodejs,
+}:
+let
+  pname = "immich-public-proxy";
+  version = "1.5.3+unstable=2024-12-02";
```

```suggestion
  version = "1.5.3-unstable-2024-12-02";
```
see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

Thanks, I didn't think to check that file. I was grepping around at existing packages and picked an outdated example to follow.